src/AddTodoForm.js

```diff
@@ -1,11 +1,30 @@
-import React from "react";
+import React, {useState} from "react";
+
+
+let AddTodoForm = ({onAddTodo}) => {
+
+  const[inputText, setInputText] = useState("");
```

Comment:
I like how you're thinking! But I feel that an extra useState in this data flow could makes things confusing to maintain. I'd recommend not using this useState, and use the onAddTodo prop instead.

Reply:
Thanks for your comments Mark.

Noted.